mne/viz/topomap.py

```diff
@@ -70,7 +70,7 @@ def _prepare_topo_plot(inst, ch_type, layout):
         # change names so that vectorview combined grads appear as MEG014x
         # instead of MEG0142 or MEG0143 which are the 2 planar grads.
         ch_names = [ch_names[k][:-1] + 'x' for k in range(0, len(ch_names), 2)]
-
+    pos = np.array(pos)[:, :2]  # 2D plot, otherwise interpolation bugs
```

Comment:
it apparently has 4 columns. Whaterv is going on, here only 2 of them should be used.